Resources/public/js/views/fields/ez-tweet-editview.js

```diff
@@ -0,0 +1,109 @@
+YUI.add('ez-tweet-editview', function (Y) {
+    "use strict";
+    /**
+     * Provides the field edit view for the Tweet (eztweet) fields
+     *
+     * @module ez-tweet-editview
+     */
+    Y.namespace('eZ');
+
+    var FIELDTYPE_IDENTIFIER = 'eztweet',
+        VALIDATION_PATTERN = /^https?:\/\/twitter.com\/([^\/]+)\/status\/[0-9]+$/;
+
+    /**
+     * Tweet edit view
+     *
+     * @namespace eZ
+     * @class TweetEditView
+     * @constructor
+     * @extends eZ.FieldEditView
+     */
+    Y.eZ.TweetEditView = Y.Base.create('tweetEditView', Y.eZ.FieldEditView, [], {
+        events: {
+            '.ez-tweet-input-ui input': {
+                'blur': 'validate',
+                'valuechange': 'validate'
+            }
+        },
+
+        /**
+         * Validates the current input of tweet
+         *
+         * @method validate
+         */
+        validate: function () {
+            var validity = this._getInputValidity();
+
+            if ( validity.typeMismatch || !this._isValidUrl() ) {
```

Comment:
For the record, instead of implementing a `_isValidUrl` method, you could use the `pattern` attribute on the input and completely rely on the validity object to validate the input (that said, the way it's done now is fine as well).